src/components/bs5/tag/tag.stories.js

```diff
@@ -2,6 +2,60 @@
 import { Tag } from './Tag.js';
 import defaultdata from './tag.data.json';
 
+const sizes = {
+  "tag-small": "Small",
```

Comment:
So these are the only allowed sizes/ emphasis and types?

Reply:
Yes, that's correct.